Taskfile.yml

```diff
@@ -0,0 +1,10 @@
+version: "3"
```

Comment:
What should be this file for?

Reply:
Oops, forgot to avoid including that! It's for another tool called `taskfile`. An alternative to using `make`. Pretty handy, but I forget not everyone is using it yet :smile: